notebooks/official/feature_store/online_feature_serving_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<p>Replace this whole block with:</p><p> </p><p>ENTITY_ID = "WO-9702714-A3" # @param {type: "string"}</p><p>country_filter = NearestNeighborQuery.StringFilter(</p><p>    name = "country",</p><p>    allow_tokens = ["WIPO (PCT)"], # please try different allow tokens</p><p>    deny_tokens = ["United States"] # please try different deny tokens</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Changed.</p>